e2e/nomostest/config_sync.go

```diff
@@ -979,6 +973,24 @@ func setupCentralizedControl(nt *NT, opts *ntopts.New) {
 	}
 }
 
+func validateRootSyncsExist(nt *NT, opts *ntopts.New) error {
+	var err error
+	for rsName := range opts.RootRepos {
+		err = multierr.Append(err,
+			nt.Validate(rsName, configsync.ControllerNamespace, &v1beta1.RootSync{}))
+	}
+	return nil
```

Comment:
return `err`

ditto for function below

Reply:
fixed